src/acvp.c

```diff
@@ -1255,6 +1255,7 @@ ACVP_RESULT acvp_process_tests (ACVP_CTX *ctx) {
      * in the regisration response.  Process each vector set and
      * return the results to the server.
      */
+	rv = ACVP_NO_CTX;
```

Comment:
Similar to the other PR... maybe if there was an error code to handle invalid or missing args and then you can check for vs_entry == NULL instead? NO_CTX could be misleading

Reply:
That makes sense. I added the changes you suggested. Let me know if that looks good to you.